inginious/frontend/plugins/contact_page/pages/api/contact_page.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of UNCode. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+
+""" Contact page """
+
+import web
+import requests
+import json
+from inginious.frontend.pages.utils import INGIniousPage
+from inginious.frontend.plugins.contact_page.pages.api.slack_url_error import SlackURLError
+
+subject_text = {
```

Comment:
I think this is a local variable, right? if so, move it to the function where this is used